v21.1/set-primary-region.md

```diff
@@ -0,0 +1,233 @@
+---
+title: SET PRIMARY REGION
+summary: The SET PRIMARY REGION statement sets the primary region of a multi-region database.
+toc: true
+---
+
+<span class="version-tag">New in v21.1:</span> The `ALTER DATABASE .. SET PRIMARY REGION` [statement](sql-statements.html) sets the primary [region](multiregion-overview.html#database-regions) of a [multi-region database](multiregion-overview.html).
+
+{% include enterprise-feature.md %}
```

Comment:
Same as above re: did you mean "{% include {{page.version.version}}/misc/enterprise-features.md %}"  ?